tests/test_upstage_ocr_model.py

```
@@ -0,0 +1,417 @@
+        model = UpstageOcrModel.__new__(UpstageOcrModel)
+        model.enabled = True
+        model.options = options
+        model.scale = 1
+        model.api_key = "test-key"
+        model.api_endpoint = options.api_endpoint
+        model.model = options.model
+        model.timeout = options.timeout
+    return model
+
+
+def _ocr_rect(l=0.0, t=0.0, r=786.0, b=256.0):
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)